spark3/src/test/java/org/apache/iceberg/spark/sql/TestDeleteFrom.java

```diff
@@ -41,6 +43,7 @@ public void removeTables() {
 
   @Test
   public void testDeleteFromUnpartitionedTable() {
+    Assume.assumeTrue(Spark3VersionUtil.isSpark30());
```

Comment:
Can you please add context when disabling tests to explain why Spark 3.0 is needed?

Reply:
I'll add an explanatory comment.